djstripe/exceptions.py

```diff
@@ -1,14 +1,28 @@
 # -*- coding: utf-8 -*-
+"""
+.. module:: djstripe.exceptions.
+
+   :synopsis: dj-stripe Exceptions.
+
+.. moduleauthor:: @kavdev
+
+"""
 from __future__ import unicode_literals
 
 
 class MultipleSubscriptionException(Exception):
+    """Raised when Customer has multiple Subscriptions."""
+
     pass
 
 
 class StripeObjectManipulationException(Exception):
+    """Raised when Cards are manipulated not through a Customer instance."""
```

Comment:
"""Raised when an attempt to manipulate a non-standalone stripe object is made not through its parent object."""